src/main/java/org/commcare/formplayer/objects/SerializableFormSession.java

```diff
@@ -11,24 +12,60 @@
 @Entity
 @Table(name = "formplayer_sessions")
 public class SerializableFormSession implements Serializable{
+    @Id @GeneratedValue(generator="system-uuid")
+    @GenericGenerator(name="system-uuid", strategy = "uuid")
     private String id;
+
+    @Column(name="instancexml")
```

Comment:
normally we wouldn't need to annotate all these fields but since we aren't following the configured naming strategy we have to supply the column names manually. I considered writing a custom naming strategy but we aren't even consistent. Also I think I'd rather just rename the columns in the DB later.